src/components/media-types/glb/index.js

```diff
@@ -7,6 +7,7 @@ export const GLBComponent = ({
   preview,
   onDetailView,
   displayView,
+  objkt,
```

Comment:
What does this refer to?

Reply:
It's the objkt ID. It's the same as with other existing components.

Reply:
Not sure to understand, can't we pass an object with more info?
I'll look into it when I have time, but again, to avoid having to add 10 new parameters to 10 files each time we want to pass something new to a function, not sure if it's clear I'll make a proposition

Reply:
Well, that's the pattern followed by the existing components. Happy to pass the whole query result down in case other info might be needed in the future.

Reply:
I would vote for that!
If not let's rename it to objktID to make it clearer

Reply:
I looked at the code a bit. It seems that the queries that drive these components' data are not the same. They are also declared in various places. Maybe, for now, go with your suggestion to rename the variables to objktID, otherwise, I fear the changes, which have little benefit for now, will make any code refactoring unnecessarily complicated.

Reply:
Yep, that's one of the big pains of the current codebase, lots of duplicated (slightly different, I guess by different devs) logic!